honeybee_schema/radiance/modifierset.py

```diff
@@ -0,0 +1,173 @@
+"""ModifierSet Schema"""
+from pydantic import Field, constr
+from typing import List, Union
+
+from .._base import IDdBaseModel
+from .modifier import ModifierBase, Void
+
+
+class BaseModifierSet(IDdBaseModel):
+    """Base class for the sets assigned to Faces."""
+
+    # TODO: What to use for default? Void or None?
+    exterior_modifier: ModifierBase = Field(
+            default=Void(),
+            description='A radiance modifier object for faces with an Outdoors boundary '
+                        'condition.'
+        )
+    # TODO: What to use for default? Void or None?
+    interior_modifier: ModifierBase = Field(
```

Comment:
This is going to create so many headaches if people end up sending us a `ModiferBase` object that has no meaning in the Radiance world. The `ModifierBase` should be virtually hidden from the perspective of the OpenAPI specification and all of these places where you have used a `ModifierBase` should be replaced with:
```
Union[Plastic, Glass, BSDF, Glow, Light, Trans]
```

Reply:
Along these lines, I don't think we should accept `Void` for any of the modifiers here. The only purpose that Void serves is to note when a modifier has no modifier. The more that I think about it, maybe we should just use `None` for all of these places where we use `Void` right now.